bucket/winauth.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "3.6.2",
+    "description": "Portable and open-source authenticator.",
```

Comment:
```suggestion
    "description": "Authenticator for wide variety of websites.",
```

Reply:
I also added "services", since WinAuth doesn't strictly work only for websites.